src/components/ga4-opl/template.njk

```diff
@@ -0,0 +1,17 @@
+<script {% if params.nonce%} nonce='{{ params.nonce }}'{%  endif %}>
+    window.addEventListener('load', function () {
+        window
+            .DI
```

Comment:
do you think we should add the window.DI?.appInit , like was suggested when I implemented in the Home repo , just in case it is undefined , it doesn't throw an error ?

Reply:
Just thought about something. We can't add "?" because it's a js code not typescript.
"?"